src/views/forms/AccountManagementForms/DepositForm.tsx

```diff
@@ -80,7 +79,9 @@ export const DepositForm = ({ onDeposit, onError }: DepositFormProps) => {
     errors: routeErrors,
     errorMessage: routeErrorMessage,
     isCctp,
-  } = useSelector(getTransferInputs, shallowEqual) || {};
+  } = useSelector(getTransferInputs, shallowEqual) ?? {};
+  // todo are these guaranteed to be base 10?
```

Comment:
evm chainIds are always whole numbers

Reply:
and only numbers? no 0x or anything?

Reply:
in any case, happy to leave it as it is for now since it's working